oxide/lib.go

```diff
@@ -22,73 +21,49 @@ const TokenEnvVar = "OXIDE_TOKEN"
 // HostEnvVar is the environment variable that contains the host.
 const HostEnvVar = "OXIDE_HOST"
 
+// defaultUserAgent is the user agent set for the client.
+const defaultUserAgent = "Oxide Go SDK"
```

Comment:
This can be something more dynamic (e.g., to include the version). Happy to discuss it further.

Reply:
Adding a version would be really nice! Let's grab it from here https://github.com/oxidecomputer/oxide.go/blob/main/VERSION

Reply:
I added the version manually for now. I left a `TODO` since I'd like to dynamically populate the version but I cannot use `embed` since the path contains a `..`. Perhaps a compile-time variable can be used instead? I left the `TODO` in so we can discuss it during this pull request.

Reply:
I responded in another discussion thread, but I ended up adding `oxide/version.go` for this.